homeassistant/components/balboa/fan.py

```diff
@@ -0,0 +1,102 @@
+"""Support for Balboa Spa pumps."""
+from __future__ import annotations
+
+import math
+from typing import Any, cast
+
+from pybalboa import SpaClient, SpaControl
+from pybalboa.enums import OffOnState, UnknownState
+
+from homeassistant.components.fan import FanEntity, FanEntityFeature
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.util.percentage import (
+    percentage_to_ranged_value,
+    ranged_value_to_percentage,
+)
+
+from .const import DOMAIN
+from .entity import BalboaEntity
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set up the spa's pumps."""
+    spa: SpaClient = hass.data[DOMAIN][entry.entry_id]
+    entities = [BalboaPumpFanEntity(control) for control in spa.pumps]
+    async_add_entities(entities)
```

Comment:
```suggestion
    async_add_entities(BalboaPumpFanEntity(control) for control in spa.pumps)
```

You can avoid creating a list here

Reply:
Done. (Though, the previous approach was consistent with existing code in this integration.)